package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "scripts": {
+    "start": "turbo run dev --no-cache --continue",
```

Comment:
@somethiiing can you see if `"start": "dev"` or `"start": "pnpm run dev"` works? We want to make sure that these stay in sync